eventually/tests/unittests/authentication/testview.py

```diff
@@ -95,3 +125,154 @@ def test_activation_bad_request(self):
         request = self.client.put(url, dumps({"email": "someemail@gmail.com", "password": "p1Rd"}),
                                   content_type='application/json')
         self.assertEqual(request.status_code, 404)
+
+    def test_get(self):
+        """        Positive test for retrieving objects from the database        """
+
+        user = json.dumps({'email':'mail@mail.co', 'password':'1234'})
+        response = self.client.post(reverse('login_user'), user, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+        user_json_toexpect = {'id': 100,
+                              'first_name': 'Robert',
+                              'middle_name': 'Jr.',
+                              'last_name': 'Downey',
+                              'email': 'mail@mail.co',
+                              'created_at': 1491825600,
+                              'updated_at': 1491825600,
+                              'is_active': True}
+
+        received_user_json = self.client.get('/api/v1/user/100/')
+        self.assertIsInstance(received_user_json, JsonResponse)
+
+        received_user_json = json.loads(received_user_json.content)
+        self.assertEqual(user_json_toexpect, received_user_json)
```

Comment:
maybe it is better to use `assertDictEqual`?

Reply:
Makes sense, corrected.